<ui version="4.0" >
 <class>CreateSampleShapeDialog</class>
 <widget class="QDialog" name="CreateSampleShapeDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sample Shape Definition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QGroupBox" name="shape_box" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Object Tree</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="details_box" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Details</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QScrollArea" name="details_scroll" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <property name="widgetResizable" >
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>232</width>
             <height>258</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="view_box" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>3D View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomlayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Workspace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="wksp_opt" />
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="maximumSize" >
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createBtn" >
       <property name="text" >
        <string>Create Shape</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>createBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateSampleShapeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>741</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel" >
     <x>438</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateSampleShapeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>827</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel" >
     <x>438</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>false</bool>
  </property>
  <property name="gridSnapY" >
   <bool>false</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
